build_node/utils/codenotary.py

```diff
@@ -62,6 +70,7 @@ def notarize_build_artifacts(
                 'srpm_sha256': srpm_sha256,
                 'srpm_nevra': srpm_nevra,
             })
+            os.remove(srpm_path)
```

Comment:
I believe that it will affect us before uploading artifacts in pulp, it looks like we delete SRPM after notarize

Reply:
No, it's safe, we do not delete any of produced artifacts